### PR TITLE
Update the import for logrus to use the updated username

### PR DESCRIPTION
**Pull request by mgleung**
## Description
Updating the logrus import to use the new updated user name (which is lowercased). This will prevent import errors with vendoring going forward.

Requires projectcalico/libcalico-go#484

## Todos
- [x] Tests
- [x] Documentation

## Release Note
```release-note
None required
```

### CODE REVIEW

**Comment by robbrockbank** [comment]
@mgleung Hi Matt, I've just checked in a change that has resulted in a merge conflict - could you rebase and fix the conflict please?

File: glide.lock [hunk 1]
```diff
@@ -178,7 +178,7 @@ imports:
   version: 5bd2802263f21d8788851d5305584c82a5c75d7e
 - name: github.com/satori/go.uuid
   version: b061729afc07e77a8aa4fad0a2fd840958f1942a
-- name: github.com/Sirupsen/logrus
+- name: github.com/sirupsen/logrus
```
Comment (robbrockbank) [inline glide.lock:160]: Did you regenerate this lock file?  I assume not as the hash ID has not changed.

File: glide.yaml [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+- package: github.com/sirupsen/logrus
   version: v0.11.2
 - package: github.com/coreos/etcd
   version: v3.1.1
   subpackages:
   - client
   - pkg/transport
 - package: github.com/osrg/gobgp
-  version: v1.16.0
+  version: v1.21.0
```
Comment (gunjan5) [inline glide.yaml]: nit: seems like glide figures this out but it's actually `v1.21`

File: glide.lock (hunk 1 above)
Comment (mgleung) [reply glide.lock:160]: Didn't at the time, I have now so everything should be up to date now

File: main.go [hunk 3]
```diff
@@ -887,7 +887,7 @@ func (s *Server) injectRoute(path *bgptable.Path) error {
 // linux kernel
 // TODO: multipath support
 func (s *Server) watchBGPPath() error {
-	watcher := s.bgpServer.Watch(bgpserver.WatchBestPath())
+	watcher := s.bgpServer.Watch(bgpserver.WatchBestPath(false))
```
Comment (gunjan5) [inline main.go:890]: is this to make the bool zero value explicitly `false`?

**Review by gunjan5 — COMMENTED** [review]
just one question, otherwise LGTM

File: main.go (hunk 3 above)
Comment (robbrockbank) [reply main.go:890]: @gunjan5 : not sure I follow.  there is no zero-value here since it's a fn argument not a struct initializer.

**Review by robbrockbank — APPROVED** [review]

File: main.go (hunk 3 above)
Comment (mgleung) [reply main.go:890]: With the update, the function requires a boolean to be passed in.  All of the previous functionality is enabled by default, setting that value to `true` will allow other functionality that we probably do not want to add at this time.

Comment (gunjan5) [reply main.go:890]: yeah my bad. ignore me

**Review by gunjan5 — APPROVED** [review]

**Pull request merged**